Dockerfile

```diff
@@ -58,7 +58,7 @@ COPY . /app
 RUN yarn build
 
 # =============================
-FROM registry.access.redhat.com/ubi8/nginx-120 as production
+FROM registry.access.redhat.com/ubi9/nginx-122 AS production
```

Comment:
1.22. is said to be too old in this discussion:
https://helsinkicity.slack.com/archives/C01RNQ5PV5Z/p1707122854109759. 

Can we used newer/latest?

Reply:
1.22 is the newest that redhat offers and should be ok. This was discussed a bit here: https://helsinkicity.slack.com/archives/C01RNQ5PV5Z/p1707311323479389?thread_ts=1707310835.048119&cid=C01RNQ5PV5Z